.eslintrc.js

```diff
@@ -12,12 +12,14 @@ module.exports = {
   rules: {
     ...base.rules,
     // this seems very broken atm, false positives
-    '@typescript-eslint/unbound-method': 'off'
+    '@typescript-eslint/unbound-method': 'off',
+    'sort-keys': 'off',
   },
   overrides: [...base.overrides, {
     files: ['**/*.test.*', '**/*.stories.*'],
     rules: {
-      '@typescript-eslint/ban-ts-comment': 'off'
+      '@typescript-eslint/ban-ts-comment': 'off',
+      '@typescript-eslint/no-unsafe-assignment': 'off'
```

Comment:
This kind of type safety is not necessary in tests, but can get quite furstrating - e.g. it prohibits usage of `expect.any(String)`, which returns the `any` type.